### PR TITLE
publish standards section to doc landing page

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Publish standards page
- Tweek previously unexposed excerpts
- Darken h1 because this one-off page is otherwise a relentless wall of gray. 

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [x] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-130667](https://oktainc.atlassian.net/browse/OKTA-130667)

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
@jmelberg-okta since this is a functional change.

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Not ready for review yet. Linking errors.

**Comment by mystiberry-okta** [comment]
@jmelberg-okta , can you take a look at index.html? In the "Standards" section, somehow I am adding an extra "/" to the path (for example, "http://localhost:4000/standards/SCIM//" instead of "http://localhost:4000/standards/SCIM/" , but I can't figure out how. The other sections using the same code don't behave that way.

File: _source/documentation/index.html [hunk 1]
```diff
@@ -6,6 +6,10 @@
     - "/docs/guides/"
     - "/docs/guides"
 ---
+
+<style>
+    h1 {color:#003366;}
```
Comment (jmelberg-okta) [inline _source/documentation/index.html]: If we update the color here, we should probably carry this out over all of our docs.

Comment (jmelberg-okta) [inline _source/documentation/index.html]: nit: The underline color for links when we hover is still the Okta blue, which looks odd next to the dark blue color.

File: _source/_standards/OAuth/index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: docs_page
 title: OAuth 2.0 and Okta
 excerpt: The next evolution of the OAuth protocol focuses on client developer simplicity.
 icon: /assets/img/icons/oauth.svg
-permalink: /standards/OAuth/
+permalink: /standards/OAuth/index
```
Comment (jmelberg-okta) [inline _source/_standards/OAuth/index.md]: Turns out, we needed to link directly to the index instead of the path root.

File: _source/documentation/index.html (hunk 1 above)
Comment (mystiberry-okta) [inline _source/documentation/index.html]: This is a unique page. I don't think updating the color to all h1 headings is good for the rest of the doc.

Comment (mystiberry-okta) [inline _source/documentation/index.html]: Yes, it should only be dark blue for category titles that aren't links. Will fix.

File: _source/_standards/OAuth/index.md (hunk 2 above)
Comment (mystiberry-okta) [inline _source/_standards/OAuth/index.md]: Brilliant, thank you!

**Comment by mystiberry-okta** [comment]
@jmelberg-okta or @rchild-okta , this PR is failing Travis tests--but the page is working as designed. It's a one-off, and I'm wondering if we've done something wrong or if it is an exception. Not sure how to proceed. I'm just trying to add a section and don't quite understand all the code I'm tinkering with.

**Comment by rchild-okta** [comment]
@mystiberry-okta this is a valid failure, if you scroll to the bottom of the Travis output you'll see the cause:

```bash
Found 114 files with 456 bad links.

To Fix:
1. Find the source .md or .html files - this script is run on the built .html files
2. Search in the source file for the problem links
3. Add a trailing slash, or reference the file directly.
For example, for '/blog', use either '/blog/' or '/blog/index.html'.
```

**tldr;** this is the exact situation which will result in a bad link when it's in production - i.e. if this is pushed, when you click those standards' links it will redirect to okta.github.io and not developer.okta.com.

**Comment by mystiberry-okta** [comment]
I didn't change anything except to add a new section, hence my confusion. I'll work with @jmelberg-okta , who did a few additional changes, and I'm sure we'll get it sorted. Thanks!

**Comment by mystiberry-okta** [comment]
@rchild-okta, Jordan researched and found this:

[2:36 PM] Jordan Melberg: npm start will correctly build the site, while when we run "npm run build-prod", it removes the trailing slash
[2:36 PM] Jordan Melberg: So oddly they are doing something different

**Comment by rchild-okta** [comment]
@jmelberg-okta @mystiberry-okta when I run `npm start` with your branch, it also generates a "bad" link, i.e. go to "http://localhost:4000/standards/OAuth/" and then right click on the "OpenId Connect and Okta" menu link, and then select "Inspect". It looks like it does not have the trailing slash.

One reason we need this link check is that locally, it "will work". I.e. if you click the link, it'll redirect you to the right place. However, github pages is served differently from how the jekyll dev server works, so that link will not redirect correctly when it's pushed to prod.

File: _source/documentation/index.html [hunk 3]
```diff
@@ -36,16 +40,37 @@ <h3 class="h4">{{ document.title }}</h3>
...
+         <a href="{{ document.url | remove: '/index' | append: '/' }}">
+            <h3 class="h4">{{ document.title }}</h3>
+         </a>
 	<p>{{ document.excerpt }}</p>
       </div>
     {% endfor %}
     </div>
     <!-- END Subpages Grid -->
 
+      <h1>Standards</h1>
```
Comment (jmelberg-okta) [inline _source/documentation/index.html]: How is this different than what starts on line 82?

While debugging, it didn't look like that HTML was rendering on the site.

**Comment by jmelberg-okta** [comment]
@rchild-okta 👉 Thanks for the response. The side-nav was causing the link-checker to fail -- since we where not appending the trailing slash.

@mystiberry-okta  👉  I'm not sure what this section does in `documentation/index.html`
```html
<!-- START Grid w/Standards -->
    <h1>Standards</h1>
    ...
    <!-- END Grid w/Standards -->
    {% endcomment %}
```

Modifying this section does not update any of our built files, so we may be able to remove this.

**Comment by mystiberry-okta** [comment]
@richardmateosian-okta This is almost ready, but don't hold up the release for it.

**Comment by mystiberry-okta** [comment]
So, I'm not sure how the new section ended up in comments. I will try again.
